apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -1229,6 +1246,8 @@ int wm_parse_commands(struct options *opt, int argc, char *argv[])
 		}
 		opt->crypto_type = get_crypto_type(argv[4]);
 		opt->password = argv[5];
+	} else if (strcmp(argv[2], "stats") == 0) {
```

Comment:
Is it better to use strncmp instead of strcmp? Because one of string length is fixed.

Reply:
Using only the given number of characters, this condition can succeed even if the input is wrong.